apps/console/src/features/identity-providers/components/forms/authenticators/sms-otp-authenticator-form.tsx

```diff
@@ -171,6 +177,11 @@ export const SMSOTPAuthenticatorForm: FunctionComponent<SMSOTPAuthenticatorFormP
     // SMS OTP length unit is set to digits or characters according to the state of this variable
     const [ isOTPNumeric, setIsOTPNumeric ] = useState<boolean>();
 
+    const [isEnableSMSOTP, setEnableSMSOTP] = useState<boolean>(false);
+    const [isReadOnly, setIsReadOnly] = useState<boolean>(true)
```

Comment:
```suggestion
    const [ isEnableSMSOTP, setEnableSMSOTP ] = useState<Boolean>(false);
    const [ isReadOnly, setIsReadOnly ] = useState<Boolean>(true)
```

Reply:
resolved in https://github.com/wso2/identity-apps/pull/3699/commits/89a5b7ce383f598b61ffaab02e48b0a23c2bf27a & https://github.com/wso2/identity-apps/pull/3699/commits/65575ba90cd4a4b1086935b6ef54e6d103e5d558